src/display/InteractiveTile.java

```diff
@@ -42,11 +42,19 @@ public void paintComponent(java.awt.Graphics g) {
 
     public class EventListener extends MouseInputAdapter {
         @Override
-        public void mouseClicked(MouseEvent e) {
+        public void mousePressed(MouseEvent e) {
             TilePressedEvent editorEvent = new TilePressedEvent(coords.x, coords.y, GameManager.getInstance().getEditor().getLayer());
             //if (EditorDisplay.getMode() == EditorModes.ADD)
             GameManager.getInstance().onEvent(editorEvent);
         }
+        @Override
+        public void mouseEntered(MouseEvent e) {
+            if (SwingUtilities.isLeftMouseButton(e))
+            {
+                TilePressedEvent editorEvent = new TilePressedEvent(coords.x, coords.y, GameManager.getInstance().getEditor().getLayer());
+                GameManager.getInstance().onEvent(editorEvent);
```

Comment:
To niby są dwie linijki, ale może lepiej jednak zrobić z tego funkcję i nie powtarzać kodu z mousePressed